src/SFA.DAS.ApprenticeCommitments.Api.AcceptanceTests/Steps/AddApprenticeshipSteps.cs

```diff
@@ -113,6 +113,7 @@ public void ThenTheRequestToTheInnerApiWasMappedCorrectly()
             innerApiRequest.ApprenticeshipId.Should().Be(_request.ApprenticeshipId);
             innerApiRequest.EmployerName.Should().Be(_request.EmployerName);
             innerApiRequest.EmployerAccountLegalEntityId.Should().Be(_request.EmployerAccountLegalEntityId);
+            innerApiRequest.TrainingProviderName.Should().Be("Provisional Training Provider Name");
```

Comment:
Just wondering if TrainingProviderId is supposed to be here?

Reply:
Yep, I've added it in with a dummy value like the name.